lib/engine/game/meta.rb

```diff
@@ -53,10 +54,15 @@ def title
             end
         end
 
+        def display_title
+          @display_title ||= (self::GAME_DISPLAY_TITLE || title)
+        end
+
         def full_title
           @full_title ||=
             self::GAME_FULL_TITLE ||
-            [self::GAME_SUPERTITLE, title, self::GAME_SUBTITLE].compact.join(': ')
+            self::GAME_DISPLAY_TITLE ||
+            [title, self::GAME_SUBTITLE].compact.join(': ')
```

Comment:
It's just occurred to me that with `display_title` being a thing now, `full_title` would be better described as `long_display_title`.

Would you have any issue with me renaming the `full_title` constants and variables in the code?